src/components/atoms/navbar.tsx

```diff
@@ -183,7 +185,9 @@ const styles = StyleSheet.create({
   settings: {
     flexDirection: 'column',
     justifyContent: 'flex-start',
-    alignItems: 'center'
+    alignItems: 'center',
+    padding: 12,
+    marginVertical: -12
```

Comment:
Is it in intentional that this brings the icon `12` to the left as well as increasing the hit area? Negative margin right seems to sort that out if not

Reply:
If this is to increase the hit target, is there a reason we did not use `hitSlop`?